packages/core/src/wallet/Wallet.test.ts

```diff
@@ -1,13 +1,24 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import type { AgentConfig } from '../../src'
+
 import { getAgentConfig } from '../../tests/helpers'
 
 import { IndyWallet } from './IndyWallet'
 
 describe('Wallet', () => {
-  const config = getAgentConfig('WalletTest')
-  const wallet = new IndyWallet(config)
+  let wallet: IndyWallet
+  let config: AgentConfig
+  let configWithMasterSecretId: AgentConfig
+
+  beforeEach(() => {
+    config = getAgentConfig('WalletTest')
+    configWithMasterSecretId = getAgentConfig('WalletTestWithMasterSecretId')
+    configWithMasterSecretId.walletConfig!.masterSecretId = 'customMasterSecretId'
```

Comment:
Maybe you can extend the getAgentConfig method and pass the masterSecretId to that method?

Reply:
Yeah, sure!